x/concentrated-liquidity/lp_test.go

```diff
@@ -345,7 +428,7 @@ func (s *KeeperTestSuite) TestWithdrawPosition() {
 			s.validatePositionUpdate(ctx, config.poolId, owner, config.lowerTick, config.upperTick, expectedRemainingLiquidity)
 
 			// check tick state
-			s.validateTickUpdates(ctx, config.poolId, owner, config.lowerTick, config.upperTick, expectedRemainingLiquidity)
+			s.validateTickUpdates(ctx, config.poolId, owner, config.lowerTick, config.upperTick, expectedRemainingLiquidity, concentrated_liquidity.EmptyCoins, concentrated_liquidity.EmptyCoins)
```

Comment:
We don't always  remove the entire position in the withdraw tests though right? Just confused on why we have two empty coins at the end for every test

Reply:
This is because this is unimplemented.

Issues that should address this:
- https://github.com/osmosis-labs/osmosis/issues/3997
- https://github.com/osmosis-labs/osmosis/issues/3951